webpack.config.js

```diff
@@ -5,12 +5,18 @@ var path = require('path');
 module.exports = getConfig({
   in: 'src/app.js',
   out: 'public',
-  clearBeforeBuild: true
+  clearBeforeBuild: true,
 });
 
 module.exports.node = {
   child_process: 'empty'
 }
 
-module.exports.devServer.host = '0.0.0.0'
+if(process.env.NODE_ENV === 'development'){ 
+	module.exports.devServer.host = '0.0.0.0'
```

Comment:
exports.devServer can be accessed only using web-pack-devserver from the npm script, this will fail while running the build command, so checking the environment and enabling accordingly.